packages/test/src/test/history-test.ts

```diff
@@ -1,11 +1,12 @@
-import { HttpSuccessCodes } from "@lionweb/repository-shared"
-import { RepositoryClient } from "@lionweb/repository-client"
+import {CreatePartitionsResponse, HttpSuccessCodes, ListPartitionsResponse} from "@lionweb/repository-shared"
+import {ClientResponse, RepositoryClient} from "@lionweb/repository-client"
 import { LanguageChange, LionWebJsonChunk, LionWebJsonDiff } from "@lionweb/validation"
 import { readModel } from "./utils.js"
 
-import { assert } from "chai"
-const { deepEqual, fail } = assert
+import {assert, expect} from "chai"
+const { deepEqual, fail, equal } = assert
 import sm from "source-map-support"
+import {LionWebJsonNode} from "@lionweb/validation/src/json/LionWebJson.js";
```

Comment:
In correct import, should import from "@lioweb/validation".
I fixed this already.